version1/04 - Web services.md

```diff
@@ -166,7 +166,7 @@ Content-Type: application/json
         ...,
         "virksomhetsspesifikkeMetadata": {
           "groupId": {
-            "fieldId": {"type": "string|long|double", "values": [string|long|double] },
+            "fieldId": {"type": "string|long|double|timestamp", "values": [string|long|double|timestamp] },
```

Comment:
Also has to be added in the response details on line 207:
`- field type (possible values are *string*, *long*, *double*, and *timestamp*)`

Reply:
Fixed. Good catch.